sys-apps/kexec-tools/files/kexec-auto-load

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env bash
+
+# Defaults
+LAYOUT=compat
+BOOTPART=/boot
+KNAME=kernel
+INITRD=initramfs.img
+
+instkern_state=/var/lib/installkernel
```

Comment:
Please have a look at https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s08.html

I guess its not a good idea to create files directly in `/var/lib` rather than a subdirectory.

If you feel like creating a subdir just for one file is not worth it than I think it should be in `/var/lib/misc`, which is a required directory so this is guaranteed to exit.

Reply:
Aah, dang, I had actually read that document, but I had missed the part where it said that subdirectories must be used. 

Fixed it now and put the state file in `misc/`.

> And I dont see, where you actually write to that statefile. Am I missing something?

That part is happening over at installkernel, since neither the state file nor the log is really specific to kexec:
- https://github.com/projg2/installkernel-gentoo/blob/master/hooks/systemd/99-write-log.install
- https://github.com/projg2/installkernel-gentoo/blob/master/hooks/99-write-log.install